data/department_choices.py

```diff
@@ -2,7 +2,8 @@
 
 
 def get_lib_department_from_code(department: str) -> str:
```

Comment:
```suggestion
def get_lib_department_from_code(department: str) -> str | None:
```
du coup ?
ou rajouter un return par défaut genre `return ""` si on veut que ca soit un str ?